Source/Core/Common/MemoryUtil.cpp

```diff
@@ -69,7 +69,8 @@ void* AllocateExecutableMemory(size_t size, bool low)
 	{
 		ptr = nullptr;
 #endif
-		PanicAlert("Failed to allocate executable memory. If you are running Dolphin in Valgrind, try '#undef MAP_32BIT'.");
+		PanicAlertT("Failed to allocate executable memory.\n"
+		            "If you are running Dolphin in Valgrind, try '#undef MAP_32BIT'.");
```

Comment:
While you're here, this could probably use a better description of what "try '#undef MAP_32BIT'" is supposed to mean.